module6/interviews/questions-4545.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Interview Questions
+---
+
+You can ask these questions in any order.  There are some questions each week that we know students will not be able to answer - this is by design to see how they exit a tough situation.  Keep track on notes on [this tracker](https://docs.google.com/spreadsheets/d/1eMtwW-06Wv0YAdAnZVm5k8_VFNKEKzQeq24kGxMTHbQ/edit?usp=sharing), please!
+
+
+## Week 1
```

Comment:
I think we should probably set a passing bar for these. Maybe 6/10?

Reply:
Maybe there is also a half point structure or something for not knowing the answer, but gracefully exiting and sharing something related that you do know?

Reply:
I'm a little cautious of setting a minimum score - what if students give really thorough answers that take up more time?  I don't want to penalize someone who doesn't get  through all of the questions, if their given answers are good.  Maybe we reduce the number of questions?

I do think there are points on the table for graceful exits.  I think a good graceful exit might even earn full points - there are definitely questions included that I do not expect students will be able to answer.

Reply:
Hmmm, that's a good point. That is tricky, I agree with you that we want to make sure students have time to answer thoroughly. 

I do think it's helpful to have a rubric before so students know they don't have to get every answer correct to pass, and that graceful exits get points.

But we could also say something along the lines of "You don't need to get all questions correct to pass and you can get credit for a graceful exit" then keep track of the scores and come up with a passing score after.